.github/scripts/release-pr-template.ejs

```diff
@@ -14,24 +14,9 @@ This is a release PR for **<%= crate.name %>** version **<%= version.actual %>**
 
 <% if (crate.name == "cargo-binstall") { %>
 Upon merging, this will automatically create the tag `v<%= version.actual %>`, build the CLI, and create a GitHub release with the release notes below.
-You will still need to manually publish the cargo crate:
-
-```
-$ git switch main
-$ git pull
-$ git switch --detach v<%= version.actual %>
-$ cargo publish -p <%= crate.name %>
-```
 <% } else { %>
 Upon merging, this will create the tag `<%= crate.name %>-v<%= version.actual %>`.
```

Comment:
also would be good to adjust this line and the one for the binary to say "...and will publish to crates.io"

Reply:
Done